bot/controbot/main.py

```diff
@@ -63,11 +63,8 @@ def read_contributor_commits(self, repository, contributor):
                 break
 
     def is_commit_valid(self, commit):
-        # Commit is a merge commit
-        if len(commit.parents) > 1:
-            return False
-
-        return True
+        # if commit is a merge commit, parent count will be higher than 1
+        return len(commit.parents) <= 1
```

Comment:
This was purposefully expanded to more than a single line, in case later on there will be added more rules.